persistence/src/main/scala/hmda/persistence/processing/HmdaFileValidator.scala

```diff
@@ -72,6 +79,16 @@ class HmdaFileValidator(submissionId: String) extends HmdaPersistentActor with L
       val validationStarted = ValidationStarted(submissionId)
       publishEvent(validationStarted)
       events(parserPersistenceId)
+        .filter(x => x.isInstanceOf[TsParsed])
+        .map(e => e.asInstanceOf[TsParsed].ts)
+        .map(ts => validateTs(ts, ctx).toEither)
+        .map {
+          case Right(ts) => ts
+          case Left(errors) => ValidationErrors(errors.list.toList)
+        }
+        .runWith(Sink.actorRef(self, NotUsed))
```

Comment:
So, I think this line points the outcome of this Flow at itself (it will send itself all the messages coming out, so `ts` will be sent right back here and be caught by `receiveCommand` case further down for the type `TransmittalSheet`).  The `NotUsed` isn't what the actor is listening for, it's the event the actor sends when the Sink has been completed.